src/connectors/infinite-hits/connectInfiniteHits.ts

```diff
@@ -285,7 +312,11 @@ const connectInfiniteHits: InfiniteHitsConnector = (
           );
         }
 
-        if (hasShowPrevious && uiState.page) {
+        if (!hasShowPrevious) {
+          return widgetSearchParameters;
+        }
```

Comment:
It doesn't have the expected behavior anymore. The widget has to set a default value otherwise it gets controlled by its parent. Here is an example of a structure that wouldn't behave like expected:

```js
search.addWidgets([
  hits(),
  pagination(),

  index().addWidgets([
    infiniteHits(),
  ]),
]);
```

Here is the live [example](https://flimsy-treatment.surge.sh/) (CodeSandbox doesn't work with the ES version at the moment). You can select a page inside the `pagination` it shouldn't impact the `infiniteHits` but it does. The widget doesn't have a default value anymore. We have to keep the page of `0` no matter what when the widget is mounted without refinement.